### PR TITLE
Fix path typo for editor def capitalize_properties

### PR DESCRIPTION
**Pull request by guilhermefelipecgs**

### CODE REVIEW

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -1381,7 +1381,7 @@ void EditorNode::_edit_current() {
 		return;
 	}
 
-	bool capitalize = bool(EDITOR_DEF("interface/editor/capitalize_properties", true));
+	bool capitalize = bool(EDITOR_GET("interface/inspector/capitalize_properties"));
```
Comment (akien-mga) [inline editor/editor_node.cpp]: Why is there both `interface/inspector/capitalize_properties` and `interface/editor/capitalize_properties`?

Comment (akien-mga) [reply editor/editor_node.cpp]:
```
$ rg "interface/.*/capitalize_properties"
editor/inspector_dock.cpp
556:    inspector->set_enable_capitalize_paths(bool(EDITOR_DEF("interface/editor/capitalize_properties", true)));

editor/editor_node.cpp
1384:   bool capitalize = bool(EDITOR_DEF("interface/editor/capitalize_properties", true));
4639:   EDITOR_DEF("interface/inspector/capitalize_properties", true);
```

Comment (akien-mga) [reply editor/editor_node.cpp]: Sounds like it should also be `interface/inspector` in `inspector_dock.cpp`.

File: editor/inspector_dock.cpp [hunk 2]
```diff
@@ -553,7 +553,7 @@ InspectorDock::InspectorDock(EditorNode *p_editor, EditorData &p_editor_data) {
 	inspector->set_v_size_flags(Control::SIZE_EXPAND_FILL);
 	inspector->set_use_doc_hints(true);
 	inspector->set_hide_script(false);
-	inspector->set_enable_capitalize_paths(bool(EDITOR_DEF("interface/editor/capitalize_properties", true)));
+	inspector->set_enable_capitalize_paths(bool(EDITOR_GET("interface/editor/capitalize_properties")));
 	inspector->set_use_folding(!bool(EDITOR_DEF("interface/editor/disable_inspector_folding", false)));
```
Comment (akien-mga) [inline editor/inspector_dock.cpp]: Maybe also rename this to `interface/inspector/disable_folding` while at it.

File: editor/editor_node.cpp (hunk 1 above)
Comment (guilhermefelipecgs) [reply editor/editor_node.cpp]: > Why is there both interface/inspector/capitalize_properties and interface/editor/capitalize_properties?

`interface/editor/capitalize_properties` is wrong, if you look in code, both are used in inspector.

File: editor/inspector_dock.cpp (hunk 2 above)
Comment (guilhermefelipecgs) [reply editor/inspector_dock.cpp]: Oops, It seems I forgot.

Comment (guilhermefelipecgs) [reply editor/inspector_dock.cpp]: Fixed.

File: editor/inspector_dock.cpp [hunk 3]
```diff
@@ -553,7 +553,7 @@ InspectorDock::InspectorDock(EditorNode *p_editor, EditorData &p_editor_data) {
 	inspector->set_v_size_flags(Control::SIZE_EXPAND_FILL);
 	inspector->set_use_doc_hints(true);
 	inspector->set_hide_script(false);
-	inspector->set_enable_capitalize_paths(bool(EDITOR_DEF("interface/editor/capitalize_properties", true)));
+	inspector->set_enable_capitalize_paths(bool(EDITOR_GET("interface/inspector/capitalize_properties")));
 	inspector->set_use_folding(!bool(EDITOR_DEF("interface/editor/disable_inspector_folding", false)));
```
Comment (akien-mga) [inline editor/inspector_dock.cpp]: As mentioned, I think this one should also get `s/editor/inspector/` for consistency.

Comment (akien-mga) [reply editor/inspector_dock.cpp]: (and then drop "inspector" from the property name, so `interface/inspector/disable_folding`)

Comment (guilhermefelipecgs) [reply editor/inspector_dock.cpp]: Ok, I'll change.

Comment (guilhermefelipecgs) [reply editor/inspector_dock.cpp]: Changed.

File: editor/inspector_dock.cpp [hunk 4]
```diff
@@ -553,8 +553,8 @@ InspectorDock::InspectorDock(EditorNode *p_editor, EditorData &p_editor_data) {
 	inspector->set_v_size_flags(Control::SIZE_EXPAND_FILL);
 	inspector->set_use_doc_hints(true);
 	inspector->set_hide_script(false);
-	inspector->set_enable_capitalize_paths(bool(EDITOR_DEF("interface/editor/capitalize_properties", true)));
-	inspector->set_use_folding(!bool(EDITOR_DEF("interface/editor/disable_inspector_folding", false)));
+	inspector->set_enable_capitalize_paths(bool(EDITOR_GET("interface/inspector/capitalize_properties")));
+	inspector->set_use_folding(!bool(EDITOR_DEF("interface/inspector/disable_inspector_folding", false)));
```
Comment (akien-mga) [inline editor/inspector_dock.cpp]: I *still* think the second `_inspector` is redundant... :)

Comment (guilhermefelipecgs) [reply editor/inspector_dock.cpp]: Yeah, maybe, I'll test it here and if it's not necessary I'll remove than.

Comment (guilhermefelipecgs) [reply editor/inspector_dock.cpp]: You'are referrgin to `interface/inspector/capitalize_properties` right?

Comment (akien-mga) [reply editor/inspector_dock.cpp]: No, I'm just saying that "interface/**inspector**/disable_**inspector**_folding" should be "interface/**inspector**/disable_folding".

"disable_folding" is enough if we're already in the "inspector" category, no need to restate it.

Comment (guilhermefelipecgs) [reply editor/inspector_dock.cpp]: I understood now, I fixed disable_folding to update without need to restart godot too.

**Comment by guilhermefelipecgs** [comment]
Don't merge yet, I'm doing some tests here.

File: editor/editor_node.cpp [hunk 5]
```diff
@@ -1437,6 +1438,7 @@ void EditorNode::_edit_current() {
 		if (current_obj->is_class("ScriptEditorDebuggerInspectedObject")) {
 			editable_warning = TTR("This is a remote object so changes to it will not be kept.\nPlease read the documentation relevant to debugging to better understand this workflow.");
 			capitalize = false;
+			folding = true;
```
Comment (akien-mga) [inline editor/editor_node.cpp]: Why enable folding for the debugger inspector? You typically want to see values in the debugger, and I think it's better to keep it using the same setting as the normal inspector.

Comment (guilhermefelipecgs) [reply editor/editor_node.cpp]: It's do the opposite, I'll fix this name.

Comment (guilhermefelipecgs) [reply editor/editor_node.cpp]: Now it's more legible I hope.

**Review by akien-mga — APPROVED** [review]

**Pull request merged**